include/WebPage_Adapter.h

```diff
@@ -396,26 +396,31 @@ void webSocketEvent(uint8_t num, WStype_t type, uint8_t *payload,
             G.conf = COMMAND_SET_MQTT;
             G.MQTT_State = split(payload, 3);
             G.MQTT_Port = split(payload, 6, 5);
-            const uint8_t index_start = 11;
+            uint8_t index_start = 11;
             payloadTextHandling(
                 payload, G.MQTT_Server,
                 sizeof(G.MQTT_Server) / sizeof(G.MQTT_Server[0]), index_start);
+            index_start =
+                index_start + sizeof(G.MQTT_Server) / sizeof(G.MQTT_Server[0]);
             payloadTextHandling(payload, G.MQTT_User,
                                 sizeof(G.MQTT_User) / sizeof(G.MQTT_User[0]),
```

Comment:
Warum das `/ sizeof(...[0])` hier überall?
Der erste Parameter von `payloadTextHandling()` ist explizit als `uint8_t` spezifiziert, also brauchen wir doch nur die Länge in Bytes. Es kommen hier doch nie irgendwelche andere Strukturen rein, bei denen wir die Anzahl Elemente brauchen, oder?
Also reicht das erste `sizeof()` und wir können das alles hier etwas kürzer und übersichtlicher gestalten.

Reply:
Hallo @tali, 

ich habe nun endlich wieder ein Abend gefunden mit der Wordclock zuzuwenden. Ich habe mit die Codepassage genauer angeschaut und paar Änderungen gemacht, sodass der Code nun an dieser stelle wesentlich leichter zu lesen ist. Ich würde mal sagen, so kann man das mergen, was meinst du ?